src/tabbar.h

```diff
@@ -68,6 +68,7 @@ public slots:
 private slots:
     void onTabMoved(int from, int to);
     void onCurrentChanged(int index);
+    void when_zimview_history_action_changed(QWebEnginePage::WebAction action, bool enabled);
```

Comment:
Usage of snake\_case here is inconsistent with camelCased function names above. I see that this is not the first such inconsistency but let's not fall victim to the broken windows effect. Also, probably it is better to replace 'when' with 'on'.